### DISCUSSION TITLE
How can I change the button text color?

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#4892] [Help]
![test](https://user-images.githubusercontent.com/29732308/201156090-6b65a8c9-c7a6-44b1-bf63-eecf4640e6a1.png)
![test2](https://user-images.githubusercontent.com/29732308/201156112-69392d48-f15f-42c6-9681-8c60bb19aa18.png)

**Comment by lutfifirdaus** [comment]
try run `npm run dev` or `npm run watch` when changing some css class
